servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -118,6 +118,9 @@ public InodeFile build() {
 
   private boolean mCompleted;
 
+  /** if any block of the file is lost */
+  private boolean mLost;
```

Comment:
That's a good question. Currently the recompute planner needs to now which files are lost. So this is a bit caching the result. Your proposed alternative is also viable: the caller can always pull the latest state from block master to check the status. Will change to it.